.cirrus.yml

```diff
@@ -22,11 +22,18 @@ task:
       test_script: ./script/incremental_build.sh test
     - name: analyze
       script: ./script/incremental_build.sh analyze
-    - name: build-apks+java-test
+    - name: build-apks+java-test+drive-examples
       env:
         matrix:
           PLUGIN_SHARDING: "--shardIndex 0 --shardCount 2"
           PLUGIN_SHARDING: "--shardIndex 1 --shardCount 2"
+          PLUGIN_SHARDING: "--shardIndex 2 --shardCount 2"
```

Comment:
should shardCount say 4?

Reply:
good catch, I'll see if I can get the tests to pass with just 2 shards and if not increase to 4